packages/core/src/awsexplorer/regionNode.ts

```diff
@@ -65,7 +65,7 @@ const serviceCandidates: ServiceNode[] = [
     },
     {
         serviceId: 'ec2',
-        when: () => DevSettings.instance.isDevMode(),
+        when: async () => await Experiments.instance.isExperimentEnabled('ec2RemoteConnect'),
```

Comment:
nit: `isExperimentEnabled` doesn't really have a reason to be async. Could drop the await from this line: https://github.com/aws/aws-toolkit-vscode/blob/e63079d3d3af18a3af27914e842d915f307b38b3/packages/core/src/shared/settings.ts#L734

Reply:
I changed it to avoid this, and use `.catch` to avoid a dangling promise.